### PR TITLE
Add link to JED - schema plugins category in schema plugin form

### PR DESCRIPTION
**Pull request by LadySolveig**
Pull Request for PR #42710 .

### Summary of Changes

Since not all schema types can be maintained in the core of Joomla itself, a new category was created in the JED (Joomla Extension Directory) especially for extending functionality with further plugins from the community.

The greatest strength of Joomla! is the community - the publication of additional schema plugins in the JED is a great opportunity for all of us. 

This PR embeds a note and a direct link to this category in JED directly in the form of the schema system plugin, so that all users have easy access to additional schema type plugins from the great developers in the community.

### Testing Instructions

If not already done configure the schema system plugin first.
To avoid misunderstandings, the notice is only displayed after the core function has been activated
grafik

After activation, a note with a direct link to the schema category in the JED should be displayed at the end of the form.

### Actual result BEFORE applying this Pull Request

No additional hint and link to JED in schema form.

### Expected result AFTER applying this Pull Request

Note with a direct link to the schema category in JED should be displayed at the end of the form.

grafik

grafik

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [X] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [X] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Review by wilsonge — APPROVED** [review]

**Comment by brianteeman** [comment]
If accepted shouldnt we add similar message to https://extensions.joomla.org/category/authoring-a-content/custom-fields/ etc

File: administrator/language/en-GB/plg_system_schemaorg.ini [hunk 1]
```diff
@@ -11,6 +11,7 @@ PLG_SYSTEM_SCHEMAORG_BASETYPE_OPTION_PERSON="Person"
...
 PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_DESCRIPTION_NOT_CONFIGURATED="To use the schema.org functionality, you have to configure the plugin first. Please contact an administrator of the page to get it configured."
 PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_DESCRIPTION_NOT_CONFIGURATED_ADMIN="To use the schema.org functionality, you have to configure the plugin first. Please select <a href=\"index.php?option=com_plugins&amp;task=plugin.edit&amp;extension_id=%s\" target=\"_blank\">this link to open the plugin</a>, configure and save."
+PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_EXTEND_JED_DESC="<strong>Need more Schema types?</strong> Extend with <a href=\"https://extensions.joomla.org/category/core-enhancements/schema-plugins" target=\"_blank\" rel=\"noopener noreferrer\">Schema Plugins from JED</a>."
```
Comment (brianteeman) [inline administrator/language/en-GB/plg_system_schemaorg.ini]: JED is a meaningless term and only means something to the "in-crowd"

Comment (brianteeman) [reply administrator/language/en-GB/plg_system_schemaorg.ini]: Dont you need to escape the " at the end of the url as well?

Comment (LadySolveig) [reply administrator/language/en-GB/plg_system_schemaorg.ini]: Thanks!

**Comment by LadySolveig** [comment]
> If accepted shouldnt we add similar message to https://extensions.joomla.org/category/authoring-a-content/custom-fields/ etc

Yes I think could be a good idea in other places too.

**Comment by fgsw** [comment]
I have tested this item :white_check_mark: successfully on 273afe4eb3c93e8d4c251d5b92ac3da1b6c6bf71<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42747">issues.joomla.org/tracker/joomla-cms/42747</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 273afe4eb3c93e8d4c251d5b92ac3da1b6c6bf71<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42747">issues.joomla.org/tracker/joomla-cms/42747</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42747">issues.joomla.org/tracker/joomla-cms/42747</a>.</sub>

**Comment by Razzo1987** [comment]
Thanks!

**Pull request merged**